3p/ampcontext.js

```diff
@@ -312,8 +313,20 @@ export class AbstractAmpContext {
       this.setupMetadata_(this.win_.name);
     }
   }
-}
 
+  /**
+   * Send 3p error to parent iframe
+   */
+  report3pError() {
+    this.win_.addEventListener('error', event => {
+      if (event.error) {
+        this.client_.sendMessage(MessageType.USER_ERROR, dict({
+          'error': event.error,
```

Comment:
You cannot, in general, send the error via postMessage. Would probably be better to stringify the fields you actually want.